modules/ipi-install-configuring-the-raid.adoc

```diff
@@ -18,19 +18,19 @@ The following procedure configures a redundant array of independent disks (RAID)
 
 . Create the manifests.
 
-. Modify the BMH (Bare Metal Host) file corresponding to the worker:
+. Modify the `BareMetalHost` resource corresponding to the node:
 +
 [source,terminal]
 ----
-$ vim clusterconfigs/openshift/99_openshift-cluster-api_hosts-3.yaml
+$ vim clusterconfigs/openshift/99_openshift-cluster-api_hosts-*.yaml
 ----
 +
 [NOTE]
 ====
-Because nodes with BMC type `irmc` do not support software RAID, the following RAID configuration uses hardware RAID as an example.
+Because nodes with BMC type `irmc` do not support software RAID, the following RAID configuration uses hardware RAID as an example. See the link:https://docs.openshift.com/container-platform/{product-version}/post_installation_configuration/bare-metal-configuration.html[Bare metal configuration] for details.
```

Comment:
Same comment here about removing this link and moving it to an additional resources section in the assembly as an xref.

Reply:
Fixed.